js/modules/k6/experimental/tracing/client.go

```diff
@@ -79,11 +79,16 @@ func (c *Client) Configure(opts options) error {
 		return fmt.Errorf("invalid options: %w", err)
 	}
 
+	var sampler Sampler = NewAlwaysSampledSampler()
+	if opts.Sampling != nil {
```

Comment:
```suggestion
	if opts.Sampling != nil && *opts.Sampling != 100 {
```

I feel that if we went through the trouble of actually having 2 implementations - let's use them more ;)